src/main/java/org/example/Ships/ShipsProcessor.java

```diff
@@ -0,0 +1,36 @@
+package org.example.Ships;
+
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import java.io.Serializable;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+@Slf4j
+public class ShipsProcessor {
+
+    HibernateFactory hibernateFactory = new HibernateFactory();
+    SessionFactory sessionFactory = hibernateFactory.getSessionFactory();
+
+    void dodaj(int length,  int xStart, int xEnd, int  yStart, int yEnd, int hits) {
+
+        try (
+                Session session = sessionFactory.openSession()) {
+            //zapisywanie nowego statku:
+            ShipsEntity ships = new ShipsEntity();
```

Comment:
nie wiem czy to dobre miejsce na tworzenie statuku :) processor powinien byc odpowiedzialny tylko za obsluge bazy danych, raczej powinien dostac juz stworzony statek